app/lib/features/chat_ng/layout/chat_ng_layout_builder.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:acter/common/extensions/acter_build_context.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/features/chat_ng/pages/chat_select_page.dart';
+import 'package:acter/features/chat_ng/rooms_list/widgets/rooms_list.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class ChatNgLayoutBuilder extends StatelessWidget {
+  final Widget? centerChild;
+  final Widget? expandedChild;
+
+  const ChatNgLayoutBuilder({this.centerChild, this.expandedChild, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final expanded = expandedChild;
```

Comment:
this seems unnecessary to me, a re-initialization for just better wording ?

Reply:
1 - this is a copy-paste
2 - remapping avoids the nasty `!` of `if (expandedChild != null) return expandedChild!`  which the static analysis still needs for reason unknown to me.